.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done your design however small parts of your design should follow the [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) provided on `requirement`. There `home` page part `"Add new category"` button should locate below of the page 🙏🏼 

![image](https://github.com/AnsarIbrahim/Budget-App/assets/63915024/2ad2d9eb-ebb6-4007-8275-f19a4c1f7d6e)